### PR TITLE
M61 release notes

### PR DESCRIPTION
**Pull request by infin8x**

### CODE REVIEW

File: themes/default/content/blog/pulumi-release-notes-m61/index.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/2715)
+
+### `pulumi config set` hides secret values
+
+Previously, setting a config value using `pulumi config set` would write the value to `stdout`, which could be undesirable if the config value was a secret. Now, config values that look like secrets will be hidden and the config key name will be written to `stdout` instead.
+
+[Learn more in this GitHub PR](https://github.com/pulumi/pulumi/pull/7327)
+
+### Dynamic providers in Python can now given a custom name
```
Comment (justinvp) [inline themes/default/content/blog/pulumi-release-notes-m61/index.md:384]: Should we highlight the improvements/fixes from external contributors? Maybe thanking them by name or GH username, and linking to their GH profile?

Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-m61/index.md:393]: Fantastic idea, thanks! Added.

File: themes/default/content/blog/pulumi-release-notes-m61/index.md [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+- Pulumi Service and Pulumi.com
+  - [Organization invites now send you to the correct identity provider]({{< relref "/blog/pulumi-release-notes-m61#organization-invites-now-send-you-to-the-correct-identity-provider" >}})
+
+<!--more-->
+
+## New and updated cloud providers
+
+### New Helm Release resource for Kubernetes
+
+Pulumi's current Helm Chart integration lacks support for some increasingly common advanced features in Helm charts, like support for [Helm lifecycle hooks](https://helm.sh/docs/topics/charts_hooks/) and handling sub-charts and [dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies). As Helm and its usage evolved over the years, Pulumi users using the `Chart` resource have often had to get very creative in order to get the desired functionality in their deployments.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-m61/index.md:34]:
```suggestion
Pulumi's existing Helm `Chart` resource provides support for deploying Helm charts via templating the chart and deploying it as Pulumi resources.  This means that some features of Helm that are not availabe via Helm templating do not behave quite the same using the Pulumi Helm `Chart` resource as they do when deployed via the `helm` CLI itself - for example features like [Helm lifecycle hooks](https://helm.sh/docs/topics/charts_hooks/) and handling sub-charts and [dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies). As Helm and its usage evolved over the years, Pulumi users using the `Chart` resource have noted that in many cases they would like a resource that provides the exact same behave as the `helm` CLI, but can be orchestrated as part of a Pulumi deployment.
```

**Review by komalali — APPROVED** [review]

File: themes/default/content/blog/pulumi-release-notes-m61/index.md [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+
+Now, the new [Helm Release]({{< relref "/docs/reference/pkg/kubernetes/helm/v3/release" >}}) resource provides Pulumi users more options to choose the right tool for their use-case. In contrast, to the existing Helm Chart integration, which is implemented as a Component Resource that extracts the corresponding Kubernetes resources' manifests from the chart and installs them as if they were individually specified in the Pulumi user program, the  new Helm Release resource uses the Helm SDK, bringing the full feature set of Helm directly to Pulumi users.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-m61/index.md:36]:
```suggestion
The new [Helm `Release`]({{< relref "/docs/reference/pkg/kubernetes/helm/v3/release" >}}) resource provides Pulumi users more options to choose the right tool for their use-case. In contrast to the existing Helm `Chart` integration, which is implemented as a Component Resource that extracts the corresponding Kubernetes resources' manifests from the chart and installs them as if they were individually specified in the Pulumi user program, the new Helm `Release` resource uses the Helm SDK directly to perform the deployment, providing the exact same behaviour as a corresponding `helm` CLI deployment.
```

File: themes/default/content/blog/pulumi-release-notes-m61/index.md [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Learn more in the [announcement blog post]({{< relref "/blog/full-access-to-helm-features-through-new-helm-release-resource-for-kubernetes" >}})
+
+### New providers: MinIO and Snowflake
+
+You can now use Pulumi's modern infrastructure-as-code solution to manage new resources! Our new providers for MinIO gives you access to the full range of high-performance, S3-compatible object storage offered by [MinIO](https://min.io). Likewise, our new provider for Snowflake lets you create and manage many of the data engineering, data lake, data warehouse, and other data resources in [Snowflake's](https://snowflake.com) data cloud.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-m61/index.md:363]:
```suggestion
You can now use Pulumi's modern infrastructure-as-code solution to manage new resources! Our new provider for MinIO gives you access to the full range of high-performance, S3-compatible object storage offered by [MinIO](https://min.io). Likewise, our new provider for Snowflake lets you create and manage many of the data engineering, data lake, data warehouse, and other data resources in [Snowflake's](https://snowflake.com) data cloud.
```

**Review by lukehoban — APPROVED** [review]
Couple relatively minor notes - but LGTM.

File: themes/default/content/blog/pulumi-release-notes-m61/index.md (hunk 3 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-m61/index.md:36]:
```suggestion
The new [Helm `Release`]({{< relref "/docs/reference/pkg/kubernetes/helm/v3/release" >}}) resource provides Pulumi users more options to choose the right tool for their use-case. In contrast to the existing Helm `Chart` integration, which is implemented as a Component Resource that extracts the corresponding Kubernetes resources' manifests from the chart and installs them as if they were individually specified in the Pulumi user program, the new Helm `Release` resource uses the Helm SDK directly to perform the deployment, providing the exact same behavior as a corresponding `helm` CLI deployment.
```

**Pull request merged**